als-suggestions/shared/src/main/scala/org/mulesoft/als/suggestions/plugins/aml/AMLRefTagCompletionPlugin.scala

```diff
@@ -68,7 +69,8 @@ trait AMLRefTagCompletionPlugin extends AMLCompletionPlugin {
       params.astPartBranch.stringValue.startsWith("$")
 
   private def isEmptyJsonValue(params: AmlCompletionRequest) =
-    params.actualDialect.id.equals(AsyncApi20Dialect.DialectLocation) &&
+    (params.actualDialect.id.equals(AsyncApi20Dialect.DialectLocation) ||
```

Comment:
if there is dialect-specific behaviour, it would be best to touch `AsyncApi20RefTag` instead of the general trait (so we don't add noice to other specifications which don't need to have this behaviour)
either way, I'm not sure why the `empty json value` would change between specs